### ISSUE TITLE
Stuck on prepare after types generated with no error...

### ISSUE THREAD

**Issue by prpanto** [nuxt/cli#352]
I try to install a nuxt project but it stuck after `types generated...`. I wait for long but nothing change...

```
...$ npm i

> postinstall
> nuxt prepare

ℹ Using default Tailwind CSS file                                                                                                                                      nuxt:tailwindcss 10:09:26 AM
✔ Types generated in .nuxt     
```

**Comment by shinGangan** [comment]
Hi @prpanto .

Do you know the following environmental information about this issue?
We would appreciate it if you could share this information with us, as the solution may vary depending on the version and environment.

- OS: macOS? Ubuntu? Windows 11?
- Nuxt version
- Node / Bun version
- is exist Error message?
- Other possible influences
  - eg1) Tailwind CSS is used, already set in nuxt.config.ts
  - eg2) I want to use xyz module

**Comment by shinGangan** [comment]
Hi @prpanto , thanks for the answer.
I thought it would be easier for the maintainer team to guess and answer the question if we had more information about the environment.

I have not observed a similar error on my end, so I am preparing a similar environment to verify the issue. Sorry I can't help you right away. 🙏

**Comment by heart** [comment]
I am facing with the same issue 

MacOS 15.3.2 (M1)
nuxt  3.16.1
Node v22.13.0
pnpm 8.11.0
npm 10.9.2

**Comment by heart** [comment]
I had the same problem in my Nuxt project

In my case, it was caused by a module that I wrote myself

I use chokidar to detect which files have changed

The problem is that when it runs this module (even though I run `pnpm i`)

it runs my module and chokidar is supposed to watch which files have changed
so it is a process that does not end but keeps running

That's why it does not continue and stops at
✔ Types generated in .nuxt

because there is a process that does not end

**Comment by danielroe** [comment]
@zcf0508 this is because you start but do not ever shutdown your caddy server when nuxt tries to shut down

**Comment by zcf0508** [comment]
<img width="1379" height="578" alt="Image" src="https://github.com/user-attachments/assets/531553a0-b8f7-4c78-86c6-46e184471be2" />

Thanks for your test, but I think I never start caddy server when running `nuxt prepare`.

**Comment by danielroe** [comment]
indeed. you can check nuxt.options._prepare to see if nuxt is running in pure typegen mode
